utils/restutil/view/view.go

```diff
@@ -41,20 +41,34 @@ type RESTView struct {
 }
 
 func (p *RESTView) RenderSuccessPost(w rest.ResponseWriter, r *rest.Request, id string) {
-	w.Header().Add(HttpHeaderLocation, fmt.Sprintf("./%s/%s", strings.TrimLeft(r.URL.Path, "/api/0.0.1/"), id))
+	w.Header().Add(
+		HttpHeaderLocation,
+		fmt.Sprintf(".%s/%s", strings.TrimSuffix(r.URL.Path, "/api/0.0.1/"), id),
```

Comment:
Dear reviewer, pay attention to this: I replaced `TrimLeft` with `TrimSuffix`, and modified slightly the format. Is this correct? :thinking:

Reply:
It passes its unit test :+1: